build-logic/src/main/kotlin/com/ibm/wala/gradle/java.gradle.kts

```diff
@@ -43,9 +43,9 @@ dependencies {
   "javadocSource"(sourceSets.main.get().allJava)
 }
 
-the<JavaPluginExtension>().toolchain.languageVersion.set(JavaLanguageVersion.of(11))
-
 tasks.withType<JavaCompile>().configureEach {
+  // Generate JDK 11 bytecodes; that is the minimum version supported by WALA
+  options.release.set(11)
```

Comment:
With the previous config that set the toolchain globally to 11, even on JDK 17 the WALA regression tests would run on JDK 11.  Now, JDK 11 bytecodes are always generated (even on JDK 17), and Java 11 is the language level, but tests are run using whatever JVM version was used to run Gradle.